tutorials/truth_gcr_variables.ipynb

```diff
@@ -128,6 +128,40 @@
     "plt.ylabel('$i$ [mag]')"
    ]
   },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "### Example: getting RA and Dec of all SNs whose brightest magnitude is brighter than a certain value\n",
+    "\n",
+    "In this example, you will need to first iterate over the objects in the light curve table. \n",
+    "You can do that by setting `return_iterator=True`. \n",
+    "For each object, you then find the brightest magnitude in the light curve, and see if it satisfies your criteria. \n",
+    "If so, then you record the object ID (`uniqueId`), which you then use to get RA, Dec from summery table later. "
+   ]
+  },
```

Comment:
This is great, but it would be great to add the 'opposite' task, or histogramming the average flux (or max flux) of the objects within a certain ra, dec? [Note that it isn't required for review, but would be great]